content/docs/command-reference/plots/index.md

```diff
@@ -48,13 +48,13 @@ differences between the metrics in different experiments.
 
 ### Supported file formats
 
-Continuous metrics can be organized as data series in JSON, CSV, or TSV files.
-DVC expects to see an array (or multiple arrays) of objects (usually _float
-numbers_) in the file.
+Continuous metrics can be organized as data series in JSON, YAML, CSV, or TSV
```

Comment:
Do we use "Continuous" in other places?

Reply:
let's replace with Array metrics, or plot metrics or just metrics?

Reply:
This is the last instance of the term "Continuous", it seems I missed removing it before. Changed to plot metrics now.

> The other good term we left in is "data series"